core/src/main/scala/filodb.core/store/ChunkSetInfo.scala

```diff
@@ -455,7 +456,8 @@ extends Iterator[ChunkSetInfoReader] {
     // advance window pointers and reset read index
     if (curWindowEnd == -1L) {
       curWindowEnd = start
-      curWindowStart = start - Math.max(window, 0)  // window cannot be below 0, ie start should never be > end
+      val windowStart = if (isInclusiveRange) window else window - 1
```

Comment:
What do you think about this, instead?
```suggestion
      val windowDuration = if (isInclusiveRange) window else window - 1
```